api/src/main/java/at/helpch/chatchat/api/event/PMSendEvent.java

```diff
@@ -0,0 +1,104 @@
+package at.helpch.chatchat.api.event;
+
+import at.helpch.chatchat.api.Format;
+import java.util.Optional;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+
+public class PMSendEvent extends Event implements Cancellable {
```

Comment:
all methods can be NotNull here

Reply:
yeah damn. I completely forgot methods can be marked as notnull. gonna do it tomorrow